.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
       - run:
           name: Run Tests
-          command: ./gradlew lint test
```

Comment:
gradle lint is a separate command than test.. So I corrected this as the main idea was to run tests